universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java

```diff
@@ -0,0 +1,55 @@
+package views;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.document;
+
+import j2html.tags.ContainerTag;
+import j2html.tags.DomContent;
+import j2html.tags.Tag;
+import javax.inject.Inject;
+import play.twirl.api.Content;
+
+/**
+ * Base class for all layout classes.
+ *
+ * <p>A layout class should describe the DOM contents of the head, header, nav, and footer. It
+ * should have a `render` method that takes the DOM contents for the main tag.
+ */
+public class BaseHtmlLayout extends BaseHtmlView {
+  protected final ViewUtils viewUtils;
```

Comment:
nit: can this be private?

Reply:
Changed it to `protected` so that derived `Layout` classes can access this. In other words it changes `Layout` classes from A to B:

A
```java
public class ApplicantLayout extends BaseHtmlLayout {
  private final viewUtils;

  @Inject
  public ApplicantLayout(ViewUtils viewUtils) {
    super(viewUtils);
    this.viewUtils = viewUtils;
  }
}
```

B
```java
public class ApplicantLayout extends BaseHtmlLayout {

  @Inject
  public ApplicantLayout(ViewUtils viewUtils) {
    super(viewUtils);
  }
}
```